client/my-sites/plugins/plans/style.scss

```diff
@@ -9,6 +9,33 @@
 	}
 }
 
+.plugins-plans .plan-features--signup .plan-features-comparison__table {
+	width: 100%;
+}
+
+.plugins-plans .plan-features--signup .plan-features-comparison__table-item {
+	border-left: none;
+}
+
+.plugins-plans .plan-features--signup .plan-features-comparison__table-item:last-of-type,
+.plugins-plans .plan-features--signup .plan-features-comparison__row .plan-features-comparison__table-item {
+	border-right: none;
+}
+
+.plugins-plans .plan-features--signup .plan-features-comparison__row:last-of-type .plan-features-comparison__table-item {
+	border-bottom: none;
+}
+
+.plugins-plans .plan-features--signup .plan-features__actions-button {
+	width: 80%;
+	margin-bottom: 20px;
+
+	&.is-current {
+		margin-top: 0;
+		width: auto;
+	}
+}
```

Comment:
Probably scope creep, can we make these buttons work without touching CSS? (there is enough going on already)

Reply:
we don't need them anymore :) 
[Fixed here](https://github.com/Automattic/wp-calypso/pull/68154/commits/82ff6ccb6de25d103c1e97c2cbe86c527a3bcdd3)